tms/utils.py

```diff
@@ -1,21 +1,77 @@
 """Utility class for frequently needed functions."""
 
-import os
+import random
+import string
 from datetime import datetime, timedelta
-from urllib.parse import urlunparse
 
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.core.files.uploadedfile import SimpleUploadedFile
-from rest_framework.test import APIClient
 
 from client.models import Client, MaleMeasurements
 from employee.models import Employee
 from order.models import Order, Task
 from product.models import Product, ProductImages
 
 
-class TestDbSetUp:
+def get_future_date(days_to_add=5):
+    """Return date in future."""
+    return datetime.today() + timedelta(days=days_to_add)
+
+
+class RandomGenerator:
+    """Functions for creating random strings."""
+
+    def get_random_alpha_string(self, length=5):
+        """Generate a random alpha string."""
+        random_string = ''.join(random.choices(string.ascii_letters, k=length))
+        return random_string
+
+    def get_random_numeric_string(self, length=13):
+        """Generate a random numeric string."""
+        random_string = ''.join(random.choices(string.digits, k=length))
+        return random_string
+
+    def get_n_unique_names(self, n):
+        """List of space separated title case strings."""
+        i = 0
+        name_set = set()
+        while True and i < n:
+            name = self.get_random_alpha_string().title() + ' ' + self.get_random_alpha_string().title()
+            if name not in name_set:
+                name_set.add(name)
+                i += 1
+        return name_set
+
+    def get_n_unique_emails(self, n):
+        """List of emails."""
+        i = 0
+        email_set = set()
+        while True and i < n:
+            email = self.get_random_alpha_string() + '@test.com'
```

Comment:
create a DUMMY_DATA_MARKER constant in management command and that should send the value '@dummytestvalues.com'